Minor grammatical updates (#122)
plantanapp/documentation: docs/audience.md

In Plant an App, Citizen Developers do, on average, 80% of the application development. They live at the intersection of the business and technical realms. On the one hand, they work to understand the business requirements and the processes involved. At the same time, they can turn them into working software that has a data model, workflows, and application screens.

> Note that a Citizen Developer rarely has experience with writing code. But they are technically savvy in the sense that they understand technology. They know concepts such as what HTTP Requests are; what the differences between back-end and front-end technologies are; or how databases are structured. Often, they even know the basics of various markup and programming languages, such as HTML, CSS, SQL, or JavaScript and can read such code for the purpose of copy-pasting and adjusting it.  

Reviewer: ```suggestion
Note that a Citizen Developer rarely has experience with writing code. But they are technically savvy in the sense that they understand technology. They know concepts such as what HTTP Requests are, what the differences between back-end and front-end technologies are, or how databases are structured. Often, they even know the basics of various markup and programming languages, such as HTML, CSS, SQL, or JavaScript, and can read such code for the purpose of copy-pasting and adjusting it.  
```

## `Software Developers`

> Plant an App has over 30 extension points, which developers can tap into, in order to add new functionalities to the platform. For example, new actions can be created as simple as implementing a C# interface and creating its specification in a JSON configuration file. Once deployed, Citizen Developers can use the new actions inside Workflows and pass in the input required for each action. Some extension points are front-end based, such as creating new views for grids or dashboards, which are Angular based.

Reviewer 2: ```suggestion
Plant an App has over 30 extension points developers can utilize to add new functionalities to the platform. For example, new actions can be created simply by implementing a C# interface and creating its specification in a JSON configuration file. Once deployed, Citizen Developers can use the new actions inside Workflows and pass in the input required for each action. Some extension points are front-end based, such as creating new views for grids or dashboards, which are Angular based.
```
